sources/be/bru/bosa-region-brussels-fr.json

```diff
@@ -0,0 +1,42 @@
+{
+    "data": "https://opendata.bosa.be/download/best/openaddress-bebru.zip",
+    "protocol": "http",
+    "coverage": {
+         "ISO 3166": {
+             "alpha2": "BE-BRU",
+             "state": "Brussels-Capital Region",
+             "country": "Belgium"
+         },
+         "country": "be",
+         "state": "brussels-capital region"
+    },
+    "conform": {
+        "format": "csv",
+        "srs": "EPSG:4326",
+        "encoding": "utf-8",
+        "accuracy": 3,
+        "number": "house_number",
+        "street": "streetname_fr",
+        "city":  "muncipality_name_fr",
+        "postcode": "postcode",
+        "id": {
+            "function": "join",
+            "fields": ["region_code","address_id"],
+            "separator": ":"
+        },
+        "unit": "box_number",
+        "lon": "EPSG:4326_lon",
+        "lat": "EPSG:4326_lat"
+    },
+    "compression": "zip",
+    "language": "fr",
+    "website": "https://opendata.bosa.be",
+    "license": {
+        "url": "https://cirb.brussels/fr/nos-solutions/urbis-solutions/licence-urbis-open-data"
```

Comment:
I can't read french, so only going by Google Translate here, but it appears to be similar to CC-BY, allowing any use, provided attribution is made.

Per the other optional tags for license at https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md

I think we should set:

```
"attribution": true,
"attribution name": <insert producer name here>
```

Reply:
ok thanks, I've added the tags